lib/Pipes/StringMapContainer.cpp

```diff
@@ -101,7 +116,7 @@ namespace revng::pipes {
 
 llvm::Error StringMapContainer::serialize(llvm::raw_ostream &OS) const {
   llvm::yaml::Output YAMLOutput(OS);
-  YAMLOutput << const_cast<std::map<MetaAddress, std::string> &>(Map);
+  YAMLOutput << const_cast<std::map<MetaAddress, String> &>(Map);
   return llvm::Error::success();
```

Comment:
Can you switch to?
```
include/revng/Support/YAMLTraits.h-void serialize(S &Stream, const T &Element) {
```
I'd like to concentrate `const_cast` in a single place.

Reply:
Done!